sdk/test/Services/DynamoDBv2/UnitTests/Custom/DynamoDBTests.cs

```diff
@@ -17,6 +17,8 @@
 
 namespace AWSSDK_DotNet35.UnitTests
 {
+    using Amazon.DynamoDBv2;
```

Comment:
nit: can you put this above `namespace` for consistency?

Reply:
changed it. thank you for the comment.